packages/deb/control

```diff
@@ -0,0 +1,32 @@
+Package: libcapstone-dev
+Source: capstone
+Version: <version-placeholder>
+Architecture: amd64
+Maintainer: Rot127 <unisono@quyllur.org>
```

Comment:
Please add you as maintainer. Since you created it and are definitely the most knowledgeable now about it.

Reply:
I don't think I can put two maintainers, and I think it is more about who maintains capstone

Reply:
Debian docs say it is the [package maintainers](https://www.debian.org/doc/debian-policy/ch-controlfields.html#s-f-maintainer).
If you want to upload this package to the Debian repo, please put you there as a maintainer.
Otherwise you can leave it like this.